app/src/pages/inside/testCaseLibraryPage/expandedOptions/mockData.js

```diff
@@ -0,0 +1,305 @@
+export const FOLDERS = [
+  {
+    name: 'Checkout flow',
```

Comment:
are we going to have ids for nodes? Do we already have some backend representation examples for this data?